packages/components/inputs/time-input/src/time-input.tsx

```diff
@@ -11,21 +10,118 @@ import { useFieldId, usePrevious } from '@commercetools-uikit/hooks';
 import TimeInputBody from './time-input-body';
 import messages from './messages';
 
+type ParsedTime = {
+  hours: number;
+  minutes: number;
+  seconds: number;
+  milliseconds: number;
+};
+
+export type TTimeInput = {
+  /**
+   * Used as HTML id property. An id is auto-generated when it is not specified.
+   */
+  id?: string;
+  /**
+   * Horizontal size limit of the input fields.
+   */
+  horizontalConstraint?:
+    | 3
+    | 4
+    | 5
+    | 6
+    | 7
+    | 8
+    | 9
+    | 10
+    | 11
+    | 12
+    | 13
+    | 14
+    | 15
+    | 16
+    | 'scale'
+    | 'auto';
```

Comment:
The initial horizontalConstraint prop option starts from 1. 

```suggestion
  horizontalConstraint?:
    | 1
    | 2
    | 3
    | 4
    | 5
    | 6
    | 7
    | 8
    | 9
    | 10
    | 11
    | 12
    | 13
    | 14
    | 15
    | 16
    | 'scale'
    | 'auto';
```

Reply:
@ddouglasz, I made a mistake. This should actually start from 3. As that was the initial values.
It's time-body-input that starts from 1. 
Sorry about that.